lib/query.dart

```diff
@@ -1,61 +1,117 @@
 part of appwrite;
 
+/// Helper class to generate query strings.
 class Query {
-    Query._();
-    
+  Query._();
+
+  /// Filter resources where [attribute] is equal to [value].
+  /// 
+  /// [value] can be a single value or a list. If a list is used
+  /// the query will return resources where [attribute] is equal
+  /// to any of the values in the list.
   static String equal(String attribute, dynamic value) =>
       _addQuery(attribute, 'equal', value);
 
+  /// Filter resources where [attribute] is not equal to [value].
+  ///
+  /// [value] can be a single value or a list. If a list is used
+  /// the query will return resources where [attribute] is equal
+  /// to any of the values in the list.
   static String notEqual(String attribute, dynamic value) =>
       _addQuery(attribute, 'notEqual', value);
 
+  /// Filter resources where [attribute] is less than [value].
+  ///
+  /// [value] can be a single value or a list. If a list is used
+  /// the query will return resources where [attribute] is equal
+  /// to any of the values in the list.
```

Comment:
I think lessThan, greaterThan, (and equal variants) only support 1 value, not array.

Reply:
updated